docs/networking/kubeovn-vpc.md

```diff
@@ -0,0 +1,667 @@
+| **Kubernetes-native networking** | Kube-OVN integrates tightly with Kubernetes, supporting CRDs, policies, etc.|
+| **Isolation and observability**  | Centralized control over IPs, ACLs, and routing through Kube-OVN            |
+
+
+## **VPC Components Overview**
+We abstract the following elements as key components within a VPC:
+
+|**Component**|**Description**|
+| :-: | :-: |
+|**VPC**|The top-level network space with a user-defined IP CIDR range|
+|**Subnet**|A subdivision of the VPC IP space; can be public or private|
+|**Route Table**|Defines traffic routing rules within and outside the VPC|
+|**Internet Gateway**|Enables outbound internet access for public subnets|
+|**NAT Gateway**|Allows private subnets to access the internet (outbound only)|
+|**Security Group**|Virtual firewall that controls inbound/outbound traffic per instance|
+|**VPC Peering / VPN**|Optional peering or hybrid connections between different VPCs or on-prem networks|
+
+*Note:You must enable `kubeovn-operator` to deploy Kube-OVN to a Harvester cluster for advanced SDN capabilities such as virtual private cloud (VPC) and subnets for virtual machine workloads.
+
+1. On the Harvester UI, go to **Advanced** > **Add-ons**.
+
+2. Select **kubeovn-operator (Experimental)**, and then select **⋮** > **Enable**.* 
+
+***VPC Creation*** 
+
+***How to Properly Map Subnets to Overlay Networks***
+
+Step 1: Create an Overlay Network
+
+1.Go to the Harvester UI.
+
+2.Navigate to Advanced > Networks.
+
+3.Click “Create” to define a new Overlay Network.
+
+- **Name example: vswitch1**\
+
+- **Type: OverlayNetwork**\
+
+4.Click Create to save.
+
+If you plan to create multiple subnets, create a separate Overlay Network for each one (e.g., vswitch2, vswitch3, etc.).
+
+Step 2: Create a Subnet and Link It to an Overlay Network
+
+1.Go to Virtual Private Cloud > Subnets.
+
+2.Click “Create”.
+
+3.Fill in the Subnet details:
+
+- **Name: vswitch1-subnet**
+
+- **CIDR: Example 172.20.10.0/24**
+
+- **Gateway IP: Example 172.20.10.1**
+
+- **Provider: Select the corresponding Overlay Network from the dropdown (e.g., default/vswitch1)**
+
+The UI will only show Overlay Networks that have not been used by other subnets — this enforces the 1:1 mapping automatically.
+
+Validation Tips:
+
+- **Once an Overlay Network is assigned to a Subnet, it cannot be reused by another.**\
+
+- **If you try to assign the same Overlay Network to multiple subnets, the system will return an error or prevent selection.**\
+
+User Reminder:
+Always ensure that each Subnet is backed by a unique Overlay Network. Mapping multiple Subnets to the same Overlay Network can cause ambiguous routing, traffic collisions, or isolation issues.
+
+- **Every new Subnet should be tied to a dedicated Overlay Network.**\
+
+- **This design also helps maintain clarity when configuring features like NAT, Private Subnet isolation, and VPC Peering.**\
+
+
+
+
+
+***Test steps:***
```

Comment:
Test steps should be added to the validating issue for QA. If you want to include examples for primary features here, you should call them out as examples instead of test steps.

Add these use cases in a specific section instead to let users understand what functions are provided. Mixing them in `VPC Components Overview` is confusing.